app.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Azimutt",
+  "description": "Next-Gen ERD: Design, Explore, Document and Analyze your database.",
+  "repository": "https://github.com/azimuttapp/azimutt",
+  "logo": "https://azimutt.app/images/logo_dark.svg",
+  "keywords": [
+    "erd",
+    "entity-relationship diagram",
+    "database tool",
+    "database schema",
+    "diagram"
+  ],
+  "stack": "container",
+  "addons": [{ "plan": "heroku-postgresql", "as": "DATABASE" }],
+  "env": {
+    "PHX_SERVER": "true",
+    "PHX_HOST": {
+      "description": "Host of the website.",
+      "value": "localhost"
+    },
+    "PORT": "4000",
+    "SECRET_KEY_BASE": {
+      "generator": "secret"
+    },
+    "FILE_STORAGE_ADAPTER": "local",
+    "AUTH_PASSWORD": "true",
+    "DATABASE_ENABLE_SSL": "true"
```

Comment:
Could be nice to also have:

- `EMAIL_ADAPTER=mailgun` with mailgun addon and the necessary env var (MAILGUN_DOMAIN, MAILGUN_API_KEY, MAILGUN_BASE_URL)
- `SKIP_ONBOARDING_FUNNEL=true`: so they try if fast without onboarding ^^
- `SKIP_EMAIL_CONFIRMATION=true`: same, faster test and don't require sending emails to test ^^
- `PUBLIC_SITE=false`: so I don't have copy of Azimutt site everywhere (I should remove it from the repo at some point)

Same for coolify ;)
++